src/backend/crossterm.rs

```diff
@@ -169,16 +165,22 @@ impl backend::Backend for Backend {
     }
 
     fn poll_event(&mut self) -> Option<Event> {
-        self.async_reader.next().map(|event| self.map_key(event))
+        match poll(Some(Duration::from_millis(10))) {
```

Comment:
Here I poll for 10 milliseconds. If there is no input for 10 millis it will return `Ok(false)`

Reply:
Do we need to poll at all? We already have some wait/polling in place outside of the backend, would it work if it was entirely non-blocking?

Reply:
We can make it a duration of 0 millis. This will almost immediately return the poll result.